packages/ember-glimmer/tests/integration/helpers/element-action-test.js

```diff
@@ -25,6 +27,54 @@ function getActionAttributes(element) {
   return actionAttrs;
 }
 
+if (isEnabled('ember-improved-instrumentation')) {
+  moduleFor('Helpers test: element action instrumentation', class extends RenderingTest {
+    ['@test action should fire interaction event with proper params']() {
+      let subscriberCallCount = 0;
+      let subscriberPayload = null;
+
+      let ExampleComponent = Component.extend({
+        actions: {
+          foo() {}
+        }
+      });
+
+      this.registerComponent('example-component', {
+        ComponentClass: ExampleComponent,
+        template: '<button {{action "foo" "bar"}}>Click me</button>'
+      });
+
+      let subscriber = subscribe('interaction.ember-action', {
```

Comment:
I think this and the teardown should be done in module setup/teardown to ensure we don't leak subscribers if this errors.